src/test/java/com/google/api/generator/engine/writer/JavaWriterVisitorTest.java

```diff
@@ -89,6 +89,29 @@ public void writeAnnotation_withDescription() {
   }
 
   /** =============================== EXPRESSIONS =============================== */
+  @Test
+  public void writeStringObjectValue() {
+    StringObjectValue s = StringObjectValue.builder().setValue("\"test\"").build();
```

Comment:
Should have picked this up in the last review, but here we want to test that the writer visitor actually writes this string. The test here would go into `ast/StringObjectValueTest.java`. In the general case, the caller wouldn't pass in `"\"test\""` unless they wanted to print `\"test\"`.